src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -5256,7 +5241,7 @@ void CheckVRFTUNodeConnections(EnergyPlusData &state, int const VRFTUNum, bool &
 {
 
     constexpr static std::string_view cTerminalUnitType("ZoneHVAC:TerminalUnit:VariableRefrigerantFlow");
-    auto &nodeID = state.dataLoopNodes->NodeID;
+    auto const &nodeID = state.dataLoopNodes->NodeID;
```

Comment:
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:5259]:(style),[constVariable],Variable 'nodeID' can be declared as reference to const